pkg/config/config.go

```diff
@@ -112,6 +112,8 @@ func getContainerDefaultNetworkIPAddress(oc *interactive.Oc, dev string) string
 
 // TestEnvironment includes the representation of the current state of the test targets and parters as well as the test configuration
 type TestEnvironment struct {
+	// TestDeployments is the list of deployments that contain pods under test.
+	TestDeployments     []configsections.Deployment
```

Comment:
to be consistent, name it deploymentsUnderTest?

Reply:
Done.